dist/images/gen_admission_keys.sh

```diff
@@ -1,9 +1,15 @@
 #!/bin/bash
 
 # Generate the CA cert and private key
-openssl req -nodes -new -x509 -keyout ca.key -out ca.crt -subj "/CN=OVN-Kubernetes Admission Controller Webhook CA"
+       openssl req -nodes -new -x509 -keyout ca.key -out ca.crt -subj "/CN=OVN-Kubernetes Admission Controller Webhook CA"
```

Comment:
gratuitous indentation?

also, if this PR is going to merge before Alexander's original one, then you should probably squash the fixes into the original commits